databroker/databroker.py

```diff
@@ -1,643 +1,22 @@
-from __future__ import print_function
-import warnings
-import six  # noqa
-from collections import deque, defaultdict
-from itertools import chain
-import pandas as pd
-import tzlocal
-from metadatastore.commands import (find_last, find_run_starts,
-                                    find_descriptors,
-                                    get_events_generator, get_events_table)
-import doct as doc
-import metadatastore.commands as mc
-import filestore.api as fs
-import logging
-import numbers
+"This module exists for back-compatability only."
 
-# Toolz and CyToolz have identical APIs -- same test suite, docstrings.
-try:
-    from cytoolz.dicttoolz import merge
-except ImportError:
-    from toolz.dicttoolz import merge
+from .broker  import Broker
+from .core import get_fields  # unused, but here for API compat
+from metadatastore.commands import _DB_SINGLETON as _MDS_SINGLETON
+from filestore.api import _FS_SINGLETON
 
+DataBroker = Broker(_MDS_SINGLETON, _FS_SINGLETON)
 
-try:
-    from functools import singledispatch
-except ImportError:
-    try:
-        # We are running on Python 2.6, 2.7, or 3.3
-        from singledispatch import singledispatch
-    except ImportError:
-        raise ImportError(
-            "Please install singledispatch from PyPI"
-            "\n\n   pip install singledispatch"
-            "\n\nThen run your program again."
-        )
-try:
-    from collections.abc import MutableSequence
-except ImportError:
-    # This will error on python < 3.3
-    from collections import MutableSequence
+get_events = DataBroker.get_events
+get_table = DataBroker.get_table
+get_images = DataBroker.get_images
+restream = DataBroker.restream
+stream = DataBroker.stream
+process = DataBroker.process
 
 
-logger = logging.getLogger(__name__)
-TZ = str(tzlocal.get_localzone())
+# TODO Add docstrings.
```

Comment:
Should just get the docstring from `DataBroker.fill_event` (programmatically even)